patches/third_party-blink-renderer-core-html-media-autoplay_policy.cc.patch

```diff
@@ -42,7 +42,7 @@ index ad5f2bf069270b6ed411e937e762d4d87580a9a7..0b6758efcc1ac6a331150e75ad4ff4ae
 +  if (!frame)
 +    return false;
 +  if (auto* settings_client = frame->GetContentSettingsClient())
-+    return settings_client->AllowAutoplay(true /* default_value */);
++    return settings_client->AllowAutoplay(false /* default_value */);
```

Comment:
cc @darkdh default true -> false?

Reply:
@pilgrim-brave I don't understand why default value == true breaks the test now but it pass in the past

Reply:
I also don't understand this. But it does.